index.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    
+    <link href="css/main.css" type="text/css" rel="stylesheet">
+    <title>Document</title>
+</head>
+<body>
+<!-- Start nav bar-->
+    <nav>         
+        <ul class="align">
+            <li>
+                <ul class="list">
+                    <li><p id="nav-text">Already have an Intuit account?</p></li>
+                    <li><p><a href="#" class="links">I forgot my User ID or Password</a></p></li>
+                </ul>
+            </li>
+            <li class="list-element"><input type="button" value="Sign in" id="sign-bttn">
+            </li>
+        </ul>
+    </nav>
+    <br>
+<!-- End nav bar-->
+
+<!-- Start main section-->
```

Comment:
Rather than only indicating the main section with an HTML comment, please use a proper `<main>` semantic tag here.